python/ray/log_monitor.py

```diff
@@ -124,7 +127,22 @@ def run(self):
         required=True,
         type=str,
         help="The IP address of the node this process is on.")
+    parser.add_argument(
```

Comment:
You could also specify `choices` here, to make it clear what levels are there. Like `choices=['debug', 'info', 'warning', 'error', 'critical'],`.